spec/features/gis_accessioning_spec.rb

```diff
@@ -108,8 +108,23 @@
     expect(page).to have_text('EPSG::3310') # form for native projection
     expect(page).to have_text('Geospatial data') # genre
     expect(page).to have_text('Cartographic dataset') # genre
-    expect(page).to have_link('View in EarthWorks') # link to Earthworks
```

Comment:
this check is duplicative of the one above that already verifies the full link with URL